mars/services/context.py

```diff
@@ -193,6 +193,20 @@ async def _fetch_chunks(self, data_keys: List[str]):
             )
         await storage_api.fetch.batch(*fetches)
 
+    def supervisor_storage_put(self, key: str, object=None):
+        return self._call(self._supervisor_storage_put(key, object))
+
+    async def _supervisor_storage_put(self, key: str, object=None):
+        storage_api = await StorageAPI.create(self.session_id, self.supervisor_address)
+        return await storage_api.put(key, object)
+
+    def supervisor_storage_get(self, key: str):
```

Comment:
The `supervisor_storage_put` and `supervisor_storage_get` is a bit confusing. In this PR, the closure is put in supervisor but get from worker.

Reply:
The service context is already messy, but the simplest way to implement this feature is to extends the context. We can wrap the storage API like this, then in the `RayExecutionContext`, they can be implemented by `ray.put` and `ray.get`.

Ray object can be evicted if no ref count exists in the cluster. But, for Mars, the UDF object should be managed by lifecycle.

```python
def storage_put(self, object=None):
    """Put the object to the storage and returns a Key."""

def storage_get(self, key) -> object:
    """Get the object from the storage"""
```